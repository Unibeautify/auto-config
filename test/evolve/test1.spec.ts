import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("test1", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `if(true){console.log({ hello: "world" });}`,
  desiredText: `if(true){console.log({ hello: "world" });}`,
});
