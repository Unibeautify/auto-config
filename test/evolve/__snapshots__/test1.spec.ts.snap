// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test1: beautifiedText 1`] = `if(true){console.log({ hello: "world" });}`;

exports[`test1: beautifiers 1`] = `
Array [
  "Prettier",
]
`;

exports[`test1: trimmedOptions 1`] = `
Object {
  "pragma_require": true,
}
`;
